src/toast/toast.js

```diff
@@ -169,6 +176,13 @@ class Toast extends React.Component<ToastPropsT, ToastPrivateStateT> {
             onMouseEnter={this.onMouseEnter}
             onMouseLeave={this.onMouseLeave}
           >
+            {typeof children === 'function' ? (
+              children({dismiss: this.dismiss})
+            ) : (
+              <ToastContent {...sharedProps} {...toastContentProps}>
```

Comment:
Does it make sense to wrap the children in both cases? `function` | React.Children